index.html

```diff
@@ -15,5 +15,107 @@
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
 
+
+<header>
+    <div id="header">
+        <div class="logo">
+            <img src="favicon.ico" alt="logo Karma"/>
+        </div>
+
+        <nav>
+            <div id="nav-bar">
+                <button class="nav-item"><strong>Meet Karma</strong></button>
+                <button class="nav-item">How it works</button>
+                <a class="nav-item" href="store.html">Store</a>
+                <button class="nav-item">Blog</button>
+                <button class="nav-item">Help</button>
+                <button class="nav-item">Login</button>
+
+            </div>
+
+        </nav>
+
+    </div>
+
+
+<div class="container">
```

Comment:
This `div` element should go inside the `main` element as this is part of the main content of the website.